src/components/bottom_bar/bottom_bar.tsx

```diff
@@ -28,10 +27,15 @@ interface Props extends CommonProps {
    * Padding applied to the bar
    */
   paddingSize?: BottomBarPaddingSize;
+
+  /**
+   * Customize the screen reader heading that helps users find this control. Default is "Page level controls".
+   */
+  title?: string;
```

Comment:
This prop name should be more specific otherwise it feels like the generic `title` attribute that you can add to any html element.  Something like 
```suggestion
  landmarkHeading?: string;
```